DigitalLearningSolutions.Web/Views/LearningSolutions/AccessibilityHelp.cshtml

```diff
@@ -2,7 +2,9 @@
 @model AccessibilityHelpViewModel
 @{
   ViewData["Title"] = "Digital Learning Solutions - Accessibility";
+  ViewData["DoNotDisplayNavBar"] = true;
 }
+<link rel="stylesheet" href="@Url.Content("~/css/learningSolutions/accessibilityAndTermsOfUse.css")" asp-append-version="true">

```

Comment:
Hmm.. could we add conditional styling in the Layout component that depends on if the DoNotDisplayNavBar is true rather than create a new stylesheet and remember to import it everywhere we use the "DoNotDisplayNavBar" flag? (might be a bit less error prone)